src/mlpack/tests/activation_functions_test.cpp

```diff
@@ -1379,3 +1428,20 @@ TEST_CASE("FlattenTSwishFunctionTest", "[ActivationFunctionsTest]")
   CheckFlattenTSwishActivationCorrect(input, desiredActivation);
   CheckFlattenTSwishDerivateCorrect(desiredActivation, desiredDerivation);
 }
+
+
+/**
+ * Basic test of the ISRU activation function.
+ */
+TEST_CASE("ISRUFunctionTest", "[ActivationFunctionsTest]")
+{
+  const arma::colvec desiredActivations("-0.89442719 0.95447998 0.97618706 \
+                                         -0.99995020 0.70710678 -0.70710678 0.89442719 0");
```

Comment:
```suggestion
  const arma::colvec desiredActivations("-0.89442719 0.95447998 0.97618706 \
                                         -0.99995020 0.70710678 -0.70710678 \
                                         0.89442719 0");
```

To be a little bit more consistent.

Reply:
Yes sorry about that!